lms/djangoapps/certificates/api.py

```diff
@@ -20,8 +20,7 @@
 from lms.djangoapps.branding import api as branding_api
 from lms.djangoapps.certificates.generation_handler import \
     generate_user_certificates as _generate_user_certificates
-from lms.djangoapps.certificates.generation_handler import \
-    is_using_certificate_allowlist as _is_using_certificate_allowlist
+from lms.djangoapps.certificates.generation_handler import _is_using_certificate_allowlist
```

Comment:
Why are we importing a method designated as private?

Reply:
Yep. Refactor went a bit far. Fixed.